src/common/patternfly.js

```diff
@@ -1,7 +1,9 @@
 import 'patternfly/dist/js/patternfly-settings';
 import 'patternfly/dist/js/patternfly-settings-charts';
+import Break from 'breakjs';
 
 const patternfly = window.patternfly;
+const layout = Break({ mobile: 0, ...patternfly.pfBreakpoints });
```

Comment:
surprised we don't have `0` in Core yet? Maybe that would be a good addition in the future... not sure if anyone is opposed to this?